src/libs/ReportUtils.ts

```diff
@@ -1609,6 +1609,33 @@ function getRoomWelcomeMessage(report: OnyxEntry<Report>): WelcomeMessage {
     return welcomeMessage;
 }
 
+function getWelcomMessage(report: OnyxEntry<Report>): WelcomeMessage {
```

Comment:
Can you please use this function in `ReportWelcomeText` for DRY principle

Reply:
@s77rt I fixed, please check again